<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compras com cartão</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Selecione o mes e o ano para pesquisar</string>
     </property>
     <widget class="QComboBox" name="txtPesquisaMes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="txtPesquisaAno">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPesquisar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icones/magnifying-glass_2703414.png</normaloff>icones/magnifying-glass_2703414.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnResetarConsulta">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Resetar consulta</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icones/reset_5578851.png</normaloff>icones/reset_5578851.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>661</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Todos os lançamentos</string>
    </property>
    <widget class="QTableWidget" name="tabelaLancamentos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>251</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data do Lançamento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data do Vencimento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Credor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Parcelas</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Classificação</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Centro de custo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Banco</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Total </string>
     </property>
     <widget class="QLabel" name="lblValor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R$ 10.020,00</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnCriarPlanilha">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Gerar Planilha</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icones/xls_9560649.png</normaloff>icones/xls_9560649.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAddAr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/oil_8084515.png</normaloff>icones/oil_8084515.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCredor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Credor</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/businessman_998372.png</normaloff>icones/businessman_998372.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddClassificacao">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Classificação</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/file_419209.png</normaloff>icones/file_419209.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCentroCusto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Centro de custo</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icones/cash_1538271.png</normaloff>icones/cash_1538271.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNovo_lan_amento">
    <property name="title">
     <string>Novo </string>
    </property>
    <addaction name="btnLancarAr"/>
    <addaction name="btnLancarCredor"/>
    <addaction name="btnClassificacao"/>
    <addaction name="btnCentroCusto"/>
   </widget>
   <addaction name="menuNovo_lan_amento"/>
  </widget>
  <action name="btnLancarAr">
   <property name="icon">
    <iconset>
     <normaloff>icones/oil_8084515.png</normaloff>icones/oil_8084515.png</iconset>
   </property>
   <property name="text">
    <string>Lançar ar</string>
   </property>
  </action>
  <action name="btnLancarCredor">
   <property name="icon">
    <iconset>
     <normaloff>icones/skills_1207390.png</normaloff>icones/skills_1207390.png</iconset>
   </property>
   <property name="text">
    <string>Credor</string>
   </property>
  </action>
  <action name="btnClassificacao">
   <property name="icon">
    <iconset>
     <normaloff>icones/file_419209.png</normaloff>icones/file_419209.png</iconset>
   </property>
   <property name="text">
    <string>Classificação</string>
   </property>
  </action>
  <action name="btnCentroCusto">
   <property name="icon">
    <iconset>
     <normaloff>icones/cash_1538271.png</normaloff>icones/cash_1538271.png</iconset>
   </property>
   <property name="text">
    <string>Centro de Custo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
